src/main/scala/firrtl/ir/IR.scala

```diff
@@ -660,25 +679,27 @@ object Formal extends Enumeration {
 }
 
 @data class Verification(
-  op:   Formal.Value,
-  info: Info,
-  clk:  Expression,
-  pred: Expression,
-  en:   Expression,
-  msg:  StringLit)
+  op:                        Formal.Value,
+  info:                      Info,
+  clk:                       Expression,
+  pred:                      Expression,
+  en:                        Expression,
+  msg:                       StringLit,
+  @since("FIRRTL 1.5") name: String = "")
```

Comment:
This is super minor, but I think it'd be nice to have a newline between the `@since` and `name` just so that the types are spaced over so much